substrait_consumer/functional/queries/sql/string_functions_sql.py

```diff
@@ -40,14 +40,22 @@
         """,
         [DataFusionProducer],
     ),
-    "ends_with": (
+    "ends_with_duckdb": (
```

Comment:
A comment here noting that providing the query in DuckDB SQL is needed to generate Substrait using the intended function could be helpful.